release.sh

```diff
@@ -48,9 +48,9 @@ package() {
 	
 	if [[ $1 == "darwin" ]]; then
 		mv "$ui/Electron.app" "$ui/Sia.app"
-		rm -r "$ui/Sia.app/Contents/Resources/default_app"
-		cp -R ../app/ "$ui/Sia.app/Contents/Resources/app/"
-		mkdir -p "$ui/Sia.app/Contents/Resources/app/Sia"
+		rm -rf "$ui/Sia.app/Contents/Resources/default_app"
+		rsync -av --progress ../ "$ui/Sia.app/Contents/Resources/app/" --exclude "release/" --exclude ".git/"
+		mkdir -p "$ui/Sia.app/Contents/Resources/app/app/Sia"
```

Comment:
So here's the thing, if we're to have one package.json and be a more traditional electron folder structure, we put ALL of our source code in electron's folder structure. That folder is called `app`. We also have a folder called `app` that siad (inside `Sia` folder) goes into.

I could rename the `app` folder in our repo to something else like `ui` to eliminate this redundancy.

I was thinking about this and the most elegant solution might be to make a clear separation in the UI where the root level of the repo is for purely development's sake. This development shell would have it's own package.json with all the devDependencies that we currently have. The `app` folder would then contain a second package.json that only has the node packages that are dependencies for the app to run (bignumber.js and jquery)

Using `electron-packager` will also do the same thing.